pints/_mcmc/_haario_ac.py

```diff
@@ -0,0 +1,103 @@
+#
+# Adaptive covariance MCMC method
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2018, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+
+
+class HaarioACMCMC(pints.GlobalAdaptiveCovarianceMCMC):
+    """
+    Adaptive Metropolis MCMC, which is algorithm 4 in [1].
```

Comment:
Ref to Haario again. And maybe some words on how this method is different? Not how the maths differs, but maybe what the intuition was or something? Not required if too hard

Reply:
Done.